clue.spec

```diff
@@ -0,0 +1,17 @@
+### RPM external clue 1.0.0
+## NOCOMPILER
+
+%define git_commit V_1_0_0
```

Comment:
@rovere , can you please also change this to
```
%define git_commit V_%(echo %{realversion} | tr '.' '_')
```

this way in future we only need to update the version in the first line :-)

Reply:
Done, thanks!

Reply:
You could also change the release tag in GitHub to `1.0.0` :-)